File: DCAnalyticsWebShell/ClientApp/src/models/SkipCondition.ts
import { Question } from "./Question"
import { Answer } from "./Answer"
import { EnumValue } from "./EnumValue"
import { DataCollectionObectTypes } from "src/enums/enums"
import { Target } from "./Target"


export class SkipCondition
{
    public OID: number
    public Key: string
    public Answer: EnumValue
    public DataCollectionObectType : DataCollectionObectTypes
    public Target: Target
    public AttributeKey: string
}

File: DCAnalyticsMobile/DCAnalyticsMobile/Models/SkipCondition.cs
using DCAnalytics;
using System;
using System.Collections.Generic;
using System.Text;

namespace DCAnalyticsMobile.Models
{
    public class SkipCondition: DataCollectionObject
    {
        public string AttributeKey { get; set; }
        public Qualifiers Qualifier { get; set; }
        public DataCollectionObectTypes DataCollectionObectType { get; set; }
        public EnumValue Answer { get; set; }
        public Target Target { get; set; }
    }
}
